python/pythonlab/neighborhood/neighborhood/support/grid_helpers.py

```diff
@@ -0,0 +1,11 @@
+class GridHelpers:
```

Comment:
Looks good, but just wondering why you created a class/static method for this instead of adding a helper function as you did for `color_helpers.py`.

Reply:
I'm still trying to figure out good python patterns 😄 I added it here because I felt it was cleaner to be able to call `GridHelpers.is_valid_2d_array` vs just calling `is_valid_2d_array`. I can go back and update `color_helpers` to follow the same pattern. What do you think?

Reply:
I think I'll leave things as-is but we can update one or the other once we decide on a pattern to go with!

Reply:
Sounds good! Browsing online, it looks like using a module for helper functions is more common. But I don't have a strong preference. https://medium.com/@ryk.kiel/python-pro-tips-organizing-and-simplifying-your-codebase-with-utilities-and-helper-functions-6a344d6a1873